app/controllers/import_controller.rb

```diff
@@ -278,6 +279,17 @@ def import_date_field(value)
     value.blank? ? nil : value
   end
 
+  def import_time_field(field, value, row_ind)
```

Comment:
We will need to also support something like this in the API. Currently the API presents the times as just numbers:
![image](https://user-images.githubusercontent.com/52747882/128098128-0e1aee75-21f5-4468-b2d1-88c4ae880687.png)

I think ideally we would want to present (and accept) values according to the FHIR time [format](https://www.hl7.org/fhir/datatypes.html#time). So something like `09:00:00` would be used here. FHIR allows for an extension to have a choice of several data types, so in this case, the `preferred-contact-method` extension will be a string when the time is something like `Morning`, but a time when the time is numerical.

Reply:
I didn't want to expand the scope too much here, so I was going to suggest leaving it as a string for now where we just use the string value of the number. But I won't object if you'd prefer to just incorporate this change now, so I'll leave it up to your preference.

Reply:
There's also the fact that with my suggestion, we're changing the way the existing preferred contact times work (Timing instead of string). Although we could still check for a `valueString` in the interest of backwards compatibility.

Reply:
Yeah +1 for not over complicating this. Tom's idea to use `Timing` sounds like the right approach, and that's a little more complexity than my original suggestion.